components/sup/src/manager/commands.rs

```diff
@@ -226,7 +226,7 @@ pub fn service_load(
         .bldr_channel
         .clone()
         .map(ChannelIdent::from)
-        .unwrap_or_default();
+        .unwrap_or_default(); // or configured_value?
```

Comment:
This looks like it should be `default` as well. This is being run on the Supervisor, and would also end up applying an environment variable value to any service being loaded, which is probably not what anybody wants. 

Confusion Quotient = _high_.